lib/cli/src/generators/SERVER/index.ts

```diff
@@ -4,6 +4,7 @@ import { copyTemplate } from '../../helpers';
 const generator: Generator = async (packageManager, npmOptions, options) => {
   baseGenerator(packageManager, npmOptions, options, 'server', {
     extensions: ['json'],
+    extraAddons: ['@storybook/addon-controls'],
```

Comment:
does server install essentials already? if so, this is redundant

Reply:
You're right it does as thats that's added by the default baseGenerator. I hadn't understood what essentials included.